test/app-luatest/gh_7288_console_flavours_vs_txn_test.lua

```diff
@@ -0,0 +1,171 @@
+local server = require('test.luatest_helpers.server')
+local console = require('console')
+local fiber = require('fiber')
+local fio = require('fio')
+local string = require('string')
+local t = require('luatest')
+
+local alocal = {
+    start = function()
+        box.cfg{memtx_use_mvcc_engine = true}
+    end,
+}
+
+local remote = {
+    start = function(self)
+        local server = server:new{
+            alias = 'default',
+            box_cfg = {memtx_use_mvcc_engine = true},
+        }
+        server:start()
+        if not self.bin then
+            self.uri = fio.abspath('.gh-7288.sock')
+            server:eval(string.format('require("console").listen("%s")', self.uri))
+        else
+            self.uri = server.net_box_uri
+        end
+        self.server = server
+    end,
+
+    stop = function(self)
+        self.server:stop()
+        self.server = nil
+        self.uri = nil
+    end,
+
+    connect = function(self, console)
+        console:send(string.format('require("console").connect("%s")', self.uri))
+    end,
+
+    disconnect = true,
+}
+
+local flavours = {
+    alocal = alocal,
```

Comment:
'alocal', because 'local' is the keyword? You can use `{['local'] = <...>}` if you want.

Reply:
But I still need to write `{['local'] = alocal}` so I'd like to use same name for both.

Reply:
Okay.